### PR TITLE
Feature/t4 t 792 restyle metadata

### PR DESCRIPTION
**Pull request by kb-smarterbalanced**
This pull request covers 3 tickets:
https://smarterbalanced.atlassian.net/browse/T4T-792
https://smarterbalanced.atlassian.net/browse/T4T-794
https://smarterbalanced.atlassian.net/browse/T4T-795

There are a few resources that are good to check:
resource/240 - has categories and multiple Related Resources
resource/241 - has no categories or Related Resources
resource/407 - has no categories and 1 Related Resource

### CODE REVIEW

**Comment by kb-smarterbalanced** [comment]
I updated the incorrect mock-data.ts - please try to build again

File: webapp/src/app/data/resource/model/related-resources.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface RelatedResources {
+    id: number;
+    title: string;
```
Comment (JJ-Muller) [inline webapp/src/app/data/resource/model/related-resources.ts:3]: @kb-smarterbalanced we are also sending the "type" or does this not need to be read in here

**Comment by kristin-dev** [comment]
Let's add it - I didn't use it for this part, but we should be consistent.

**Comment by kb-smarterbalanced** [comment]
Actually, I was using this wrong in one place, I corrected that as well.

**Review by JJ-Muller — APPROVED** [review]

**Pull request merged**